experiments/swdt/pkg/connections/ssh.go

```diff
@@ -111,7 +117,10 @@ func (c *SSHConnection) Run(args string) (string, error) {
 	// Multiline PowerShell commands over SSH trip over newlines - only first one is executed
 	args = regexp.MustCompile(`\r?\n`).ReplaceAllLiteralString(args, ";")
 	cmd := fmt.Sprintf("powershell -nologo -noprofile -c { %s }", args)
-	// TODO(mloskot): Log executed commands in --verbose mode
+
+	// TODO(mloskot): Why this is not printed?
+	klog.V(1).Infof("SSH executing command: %s", cmd)
```

Comment:
@knabben Would you have an idea why this is not outputting anything, and `klog` in the `Connect` above too?
It looks like the `-v` flag is completely ignored here, and `klog.V(1).Enabled()` is false as well.
Interestingly, `klog.V(1)...` in all other places works fine, in `cmd` package, in `config` package. My very limited klog fu is out of clues :)

Here is how I run SWDT CLI:

```powershell
.\swdt.ps1 setup -v 3 --config .\experiments\swdt\samples\mloskot\winworker.yaml
```